test/services/LibraryService.test.ts

```diff
@@ -5,31 +5,40 @@ import supertest from 'supertest';
 
 let server: Server;
 
-const Library: fhir4.Library = {
+const LIBRARY_WITH_URL: fhir4.Library = {
   resourceType: 'Library',
   type: { coding: [{ code: 'logic-library' }] },
-  id: 'library123',
+  id: 'testWithUrl',
+  status: 'active',
+  url: 'http://example.com'
+};
+
+const LIBRARY: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'test',
   status: 'active'
 };
 
 describe('LibraryService', () => {
-  beforeAll(() => {
-    server = initialize(serverConfig);
-    return setupTestDatabase([Library]);
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
+    await setupTestDatabase([LIBRARY, LIBRARY_WITH_URL]);
```

Comment:
Looks like some of the requested changes from #2 were lost in translation during a rebase. Can we follow the pattern suggested in those changes instead?

```suggestion
  beforeAll(() => {
    server = initialize(serverConfig);
    return setupTestDatabase([LIBRARY, LIBRARY_WITH_URL]);
```